src/services/lnd/index.ts

```diff
@@ -1,11 +1,20 @@
+import { LndOfflineError } from "@core/error"
 import {
   decodeInvoice,
   CouldNotDecodeReturnedPaymentRequest,
   UnknownLightningServiceError,
 } from "@domain/bitcoin/lightning"
 import { createInvoice } from "lightning"
+import { getActiveLnd } from "./utils"
+
+export const LndService = (): ILightningService => {
+  let lndAuth: AuthenticatedLnd, pubkey: string
+  try {
+    ;({ lnd: lndAuth, pubkey } = getActiveLnd())
+  } catch (err) {
+    throw new LndOfflineError("no active lnd to create an invoice")
```

Comment:
should be returned as error not thrown.
`export const LndService = (): ILightningService | LightningServiceError`